tests/sles4sap/trento/test_hana_restore_stopped.pm

```diff
@@ -37,9 +37,16 @@ sub run {
     my $prov = get_required_var('PUBLIC_CLOUD_PROVIDER');
     # vmhana01 hard-coded in place of the generic Ansible filter from $primary_host.
     # The Ansible generic filter is only valid for Ansible, here it is crm.
-    qesap_ansible_cmd(cmd => "sudo crm resource refresh rsc_SAPHana_HDB_HDB00 vmhana01",
-        provider => $prov,
-        filter => $primary_host);
+    if (check_var('ANGI', 'true')) {
```

Comment:
Same as above, no need to copy code multiple times:
```
my $hana_resource_name = get_var('ANGI') ? 'rsc_SAPHanaCtl_HDB_HDB00' : 'rsc_SAPHana_HDB_HDB00';
    qesap_ansible_cmd(cmd => "sudo crm resource refresh $hana_resource_name vmhana01",
        provider => $prov,
        filter => $primary_host);
```
Not sure if possible but I would suggest replacing hardcoded SID and instance numbers with parameters as it is done above.

Reply:
Exactly! It's a good question and if we found a job in which it's used, we could check.